Services/UI/OJS.Services.Ui.Business/Implementations/SubmissionsBusinessService.cs

```diff
@@ -363,8 +363,7 @@ public async Task ProcessExecutionResult(SubmissionExecutionResult submissionExe
 
         if (exception != null)
         {
-            submission.ProcessingComment = exception.Message + Environment.NewLine + exception.StackTrace;
-            return;
+            submission.ProcessingComment = exception.Message;
```

Comment:
I think exception.StackTrace is useful and it can be added to the ProcessingComment